src/main.cpp

```diff
@@ -3019,14 +3019,17 @@ static void NotifyHeaderTip() {
 bool ActivateBestChain(CValidationState &state, const CChainParams& chainparams, const CBlock *pblock) {
     CBlockIndex *pindexMostWork = NULL;
     CBlockIndex *pindexNewTip = NULL;
+    std::vector<std::tuple<CTransaction,CBlockIndex*,int> > txChanged;
```

Comment:
In c++11 the ugly '> >' (with a space in between) is no longer needed. Might as well fix it when you're changing it.